### ISSUE TITLE
Table of contents/Jump to a chapter

### ISSUE THREAD

**Issue by ajleon95** [susannalles/MinimalEditions#2]
I think our web edition should have a table of contents that is easily accessible (maybe as a sidebar) and allows navigation between chapters.

**Comment by rmania1** [comment]
Yes. I believe I have talked about this two classes ago. So, when I am in the middle of "Tratado segundo", I should be able to go to the beginning of "Tratado tercero" by clicking on "Tratado tercero" located on the sidebar.
